examples/neighbors/plot_nca_illustration.py

```diff
@@ -87,8 +85,8 @@ def relate_point(X, i, ax):
 for i in range(len(X)):
     ax2.text(X_embedded[i, 0], X_embedded[i, 1], str(i),
              va='center', ha='center')
-    ax2.scatter(X_embedded[i, 0], X_embedded[i, 1], s=300, c=cm.Set1(y[i]),
-                alpha=0.4)
+    ax2.scatter(X_embedded[i, 0], X_embedded[i, 1], s=300,
+                c=np.array([cm.Set1(y[i])]), alpha=0.4)
```

Comment:
This is the same as above `cm.Set1([y[[i]])`

Reply:
They are not the same `cm.Set1(y[[i]])` is a tuple while `np.array([cm.Set1(y[i])])` is an array. Using the tuple brings errors

Reply:
`y[[1]]` is a numpy array and it works while `y[1]` will return a scalar which raises the warning. There is no tuple. I tested locally and it works.

Reply:
```suggestion
                c=np.array([cm.Set1(y[[i]])]), alpha=0.4)
```